src/EncodingTypes.h

```diff
@@ -8,7 +8,8 @@ class EncodingTypes
     enum Enumeration
     {
         Thrift,
-        ByteStream
+        ByteStream,
```

Comment:
There is no Bytestream serialization, it needs to be removed, only Thrift and Json are supported.